### PR TITLE
fix: remove code cov and git hacks

### LINKED ISSUE #4: fix: istanbul branch coverage 0%
I don't think the `/* istanbul ignore if */` is working?

ref:
- https://github.com/pke/parse-app-info/blob/master/index.js#L17
- https://travis-ci.com/pke/parse-app-info/jobs/150161526#L483
- https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md

### PR DESCRIPTION
**Pull request by pke**
Using proper mocks now for testing env var code paths and non existing git
infos.

closes #4

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      "program": "${workspaceRoot}/node_modules/ava/profile.js",
+      "args": [
+        "${file}"
+      ],
+      "skipFiles": [
+        "<node_internals>/**/*.js"
+      ]
+    }
+  ]
+}
```
Comment (niftylettuce) [inline .vscode/launch.json]: Can we keep VS Code files out of the repo per chance? Everything else looks incredible. Thank you!!!!!!

**Review by niftylettuce — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (pke) [reply .vscode/launch.json]: Sure.

Comment (pke) [reply .vscode/launch.json]: Sure.

Comment (pke) [reply .vscode/launch.json]: Removed it and force pushed.

**Pull request merged**